### PR TITLE
[MRG+1] Convert remaining test_t* files to pytest.

### PR DESCRIPTION
**Pull request by QuLogic**
"remaining" meaning not covered by the other PRs.

### CODE REVIEW

**Comment by jkseppan** [comment]
Looks good to me.

File: lib/matplotlib/tests/test_triangulation.py [hunk 1]
```diff
@@ -96,16 +97,21 @@ def test_delaunay_points_in_line():
 
 def test_delaunay_insufficient_points():
```
Comment (phobson) [inline lib/matplotlib/tests/test_triangulation.py]: Could this test be parametrized or are we not doing that yet?

File: lib/matplotlib/tests/test_tightlayout.py [hunk 2]
```diff
@@ -8,7 +8,6 @@
 
 from matplotlib.testing.decorators import image_comparison, knownfailureif
 import matplotlib.pyplot as plt
-from nose.tools import assert_raises
 from numpy.testing import assert_array_equal
```
Comment (phobson) [inline lib/matplotlib/tests/test_tightlayout.py]: This is never used AFAICT

File: lib/matplotlib/tests/test_triangulation.py (hunk 1 above)
Comment (QuLogic) [inline lib/matplotlib/tests/test_triangulation.py]: Done.

**Pull request merged**